### PR TITLE
fix(releasenotes): highlight note regarding 3.11 support

### PR DESCRIPTION
**Pull request by majorgreys**
The note about lack of compatibility should be highlighted.

**Current**

image

**Preview**

image

### CODE REVIEW

File: releasenotes/notes/python311-tracer-a7077c0e461622d2.yaml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ---
 prelude: >
-    Initial library support has been added for Python 3.11. Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for Continuous Profiler and Dynamic Instrumentation will be added in a future release.
-    For full Python 3.11 support status please see: https://github.com/DataDog/dd-trace-py/issues/4149
+  Initial library support has been added for Python 3.11.
+
+
+  .. note::
+    Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for them will be added in a future release. For a full status, see the `Support Python 3.11 #4149<https://github.com/DataDog/dd-trace-py/issues/4149>`_ issue on GitHub.
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/python311-tracer-a7077c0e461622d2.yaml:7]:
```suggestion
    Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for them will be added in a future release. To track the status, see the `Support Python 3.11 #4149<https://github.com/DataDog/dd-trace-py/issues/4149>`_ issue on GitHub.
```

**Review by Kyle-Verhoog — COMMENTED** [review]
one nit, otherwise lgtm

**Review by Yun-Kim — DISMISSED** [review]

**Review by ZStriker19 — DISMISSED** [review]

File: releasenotes/notes/python311-tracer-a7077c0e461622d2.yaml [hunk 2]
```diff
@@ -1,7 +1,12 @@
 ---
 prelude: >
-    Initial library support has been added for Python 3.11. Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for Continuous Profiler and Dynamic Instrumentation will be added in a future release.
-    For full Python 3.11 support status please see: https://github.com/DataDog/dd-trace-py/issues/4149
+  Initial library support has been added for Python 3.11.
+
+
+  .. note::
+    Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for them will be added in a future release. For a full status, see the `Support Python 3.11 <https://github.com/DataDog/dd-trace-py/issues/4149>`_ issue on GitHub.
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/python311-tracer-a7077c0e461622d2.yaml:7]:
```suggestion
    Continuous Profiler and Dynamic Instrumentation are not yet compatible and must be disabled in order to use the library with Python 3.11. Support for them will be added in a future release. To track the status, see the `Support Python 3.11 <https://github.com/DataDog/dd-trace-py/issues/4149>`_ issue on GitHub.
```

**Comment by majorgreys** [comment]
@Mergifyio backport 1.6

**Review by Yun-Kim — APPROVED** [review]

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**